guidelines/responses.md

```diff
@@ -18,13 +18,15 @@ In case of success, the HTTP status code is normally 200 and the content contain
   * Error caused by usage of invalid ClientToken, AccessToken, or you may not have the necessary permission to use the endpoint.
 * **403 Forbidden**
   * Server error that should be reported to the end user of the client app. Happens for example when the server-side validation fails or when a business-logic check is violated.
+* **404 Not found**
+  * The server cannot find the resource requested by the client app. Verify the URL and request method.
 * **408 Request Timeout**
   * Error caused by heavy request that takes too long to process (typically tens of seconds). To get around this, request data in smaller batches. For more information, see [Request timeouts](requests.md#request-timeouts).
 * **409 Conflict**
   * The request cannot be completed because the data has changed. To get around this error, try refreshing the data on the client and resubmitting the request.
 * **429 Too Many Requests**
   * Error caused by too many requests sent in a given amount of time. Response contains `Retry-After` header indicating how long the user agent should wait before making a follow-up request. For more information, see [Request limits](requests.md#request-limits).
-* **500 Internal Server Error**
+* **5xx Internal Server Error**
```

Comment:
@jnv at the end, I've changed this to `5xx`, do you agree with this change or should we keep it as 500? 🤔

Reply:
I'd keep 500 and add 5xx as a separate entry, with more detailed explanation. Or alternatively, since 5xx will have the same explanation, how about this:

```suggestion
* **5xx Internal Server Error** and other **5xx** response codes codes
```

Reply:
I like it... should it be mention in the changelog? 🤔